control-plane-operator/controllers/hostedcontrolplane/snapshotcontroller/assets/04_serviceaccount-hypershift.yaml

```diff
@@ -0,0 +1,7 @@
+# *** AUTOMATICALLY GENERATED FILE - DO NOT EDIT ***
+apiVersion: v1
```

Comment:
This yaml files don't need to be suffixed with -hypeshift since this is the hypershift repo already.

Reply:
We will copy the yaml files from openshift/cluster-csi-snapshot-controller-operator (see https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/125) and they do have -hypershift in the name. I kept the name to make it really obvious what file should be copied.